constraint/config/crds/templates.gatekeeper.sh_constrainttemplates.yaml

```diff
@@ -42,6 +46,8 @@ spec:
                           type: array
                       type: object
                     validation:
+                      properties:
```

Comment:
type: object (this can be done with a controller-gen tag)

Reply:
Done